department-of-veterans-affairs/va.gov-cms
# VACMS-3688: Covid Vaccine system and facility services

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3688

## Testing done


## Screenshots


## QA steps

- Navigate to preview environment
- [x] Enable covid-19 taxonomy term: /taxonomy/term/321/edit?destination=/admin/structure/taxonomy/manage/health_care_service_taxonomy/overview
- Navigate to migrations page here: /admin/structure/migrate/manage/vamc/migrations
- [x] Confirm that the following migrations are visible with the following attributes:
  - [x] **MIGRATION | MACHINE NAME | STATUS | TOTAL | IMPORTED | UNPROCESSED | MESSAGES | LAST IMPORTED |** 
          _Node - VAMC Facility Health Service. | facility_health_care_service_covid | Idle | 37 | 0 | 37 | 0 |   |  
          Node - VAMC Health System. | health_care_region_page | Idle | 74 | 70 | 4 | 0 | 2020-12-15 17:13:38
          Node - VAMC System Health Service keyed by system. | regional_health_care_service_covid | Idle | 37 | 0 | 37 | 0_

-  Select EXECUTE for the following script **facility_health_care_service_covid** and confirm the following system message:
- [x] Processed 4 items (4 created, 0 updated, 0 failed, 0 ignored) - done with 'health_care_region_page'
        Processed 37 items (37 created, 0 updated, 0 failed, 0 ignored) - done with 'regional_health_care_service_covid'
        Processed 37 items (37 created, 0 updated, 0 failed, 0 ignored) - done with 'facility_health_care_service_covid'

- Navigate to the content list page: /admin/content
- [x]  Confirm the creation of the 4 missing VAMC Systems, VAMC System Health Service, VAMC Facility Health Service based on the sample data below:

### VAMC System Pages

`VISN;VAMC system plain language name;VAMC system official name;Meta title tag;Meta Description;GovDelivery ID for Emergency Updates email;GovDelivery ID for News and Announcements;Facebook URL;Link text;Twitter URL;Link text;Flickr URL;link text;Instagram URL;link text;URL alias`

`5;'VA Maryland health care';'VA Maryland Healthcare System';'VA Maryland health care | Veterans Affairs';'Find a health facility near you at [VA System name], and manage your health online. Our health care teams are deeply experienced and guided by the needs of Veterans, their families, and caregivers.';'USVHA_400';'USVHA_87';'https://www.facebook.com/MDVAHCS';'VA Maryland Healthcare System Facebook page';'https://twitter.com/mdvagov';'VA Maryland Healthcare System at Twitter';'';'';'';'';'/maryland-health-care/'`

`5;'VA Washington DC health care';'VA Washington DC Healthcare System';'VA Washington DC health care | Veterans Affairs';'Find a health facility near you at [VA System name], and manage your health online. Our health care teams are deeply experienced and guided by the needs of Veterans, their families, and caregivers.';'USVHA_401';'USVHA_89';'https://www.facebook.com/WashingtonDCVAMC';'VA Washington DC Healthcare System Facebook page';'https://twitter.com/DCVAMC';'VA Washington DC Healthcare System at Twitter';'';'';'';'';'/washington-dc-health-care/'`

`8;'VA Bedford health care';'VA Bedford Healthcare System';'VA Bedford health care | Veterans Affairs';'Find a health facility near you at [VA System name], and manage your health online. Our health care teams are deeply experienced and guided by the needs of Veterans, their families, and caregivers.';'USVHA_370';'USVHA_116';'https://www.facebook.com/BedfordVAHospital';'VA Bedford Healthcare System Facebook page';'';'';'';'';'';'';'/bedford-health-care/'`

`8;'VA Connecticut health care';'VA Connecticut Healthcare System';'VA Connecticut health care | Veterans Affairs';'Find a health facility near you at [VA System name], and manage your health online. Our health care teams are deeply experienced and guided by the needs of Veterans, their families, and caregivers.';'USVHA_375';'USVHA_112';'https://www.facebook.com/VAConnecticut';'VA Connecticut Healthcare System Facebook page';'';'';'';'';'';'';'/connecticut-health-care/'`

### VAMC System and Facility Health Care Services
`Id;Facility Name;System Name;System ID;System description of services;Facility description of services`
`518;'Edith Nourse Rogers Memorial Veterans' Hospital';'VA Bedford health care';'VAHCS-MA-01';'';''`
`689;'West Haven VA Medical Center';'VA Connecticut health care';'VAHCS-CT-01';'';''`
`630;'Manhattan VA Medical Center';'VA New York Harbor health care';'VAHCS-NY-08';'';''`
`528;'Buffalo VA Medical Center';'VA Western New York health care';'VAHCS-NY-04';'';''`
`642;'Corporal Michael J. Crescenz VA Medical Center';'VA Philadelphia health care';'VAHCS-PA-06';'';''`
`646;'Pittsburgh VA Medical Center-University Drive';'VA Pittsburgh health care';'VAHCS-PA-07';'';''`
`512;'Baltimore VA Medical Center';'VA Maryland health care';'VAHCS-MD-01';'';''`






### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
